.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,25 @@
+## Description
+
 <!--
-    If this is a security-related patch stop immediately!
-    See http://docs.django-cms.org/en/latest/contributing/development-policies.html
+If this is a security issue stop right here and follow our documentation:
+http://docs.django-cms.org/en/latest/contributing/development-policies.html#reporting-security-issues
 -->
 
+## Related resources
 
-### Summary
-
-Fixes #
-
-
-### Links to related discussion
-
+<!--
+Add here links to existing issues or conversation from GitHub
+or any other resource.
+-->
 
-### Proposed changes in this pull request
+* #...
+* #...
 
+## Checklist
 
-### General checklist
+<!--
+Please check the following items before submitting, otherwise, your pull request will be closed.
+-->
 
-* [ ] I have updated the CHANGELOG.txt
-* [ ] I have created backports if necessary
-* [ ] I have updated the documentation and/or amended the upgrade section
-      if necessary
+* [x] I have opened this pull request against ``develop``
```

Comment:
Why is this one pre-checked?

Reply:
to indicate how to check something, there were several issues where it was either [* ], [check] or [. ]